components/Navbar.jsx

```diff
@@ -13,47 +13,116 @@ export default function Navbar() {
 
     return (
         <div>
-            <nav className="sticky flex flex-wrap p-6 items-center justify-between">
+            <nav className="sticky flex flex-wrap p-6 md:px-0 items-center justify-between">
```

Comment:
md:px-0 ?
looks very weird when it sticks to the absolute side of the screen.

Reply:
![image](https://user-images.githubusercontent.com/79566582/230765899-6bb988d3-af1a-4f38-9c6a-20b86db5fab6.png)

Reply:
Somry.. changing to `lg:px-0`